web/app/assets/locales/locale-tr.js

```diff
@@ -32,7 +32,7 @@
         pending: "sırada %(blocks)s blok var"
     },
     propose: "teklif et",
-    cancel: "iptal et",
```

Comment:
There was one error like this in an earlier PR, not sure if it was in the same place